randovania/games/prime1/logic_database/Phendrana Drifts.txt

```diff
@@ -704,7 +704,7 @@ Extra - aabb: [-71.61841, -514.37103, 24.210918, 74.98186, -419.0276, 94.39217]
           Scan Visor and Combat/Scan Dash (Advanced)
   > Door to Specimen Storage
       Any of the following:
-          Space Jump Boots
+          Space Jump Boots or Combat/Scan Dash (Beginner)
```

Comment:
Combat/Scan Dash (Beginner) and Scan Visor
Combat/Scan Dash (Intermediate) and X-Ray Visor and Before Thardus